com.github.muriloventuroso.pdftricks.json

```diff
@@ -25,10 +32,9 @@
                 "/share"
             ],
             "sources": [{
-                "__comment": "Must be version 1.2.x",
-                "type": "archive",
-                "url": "https://downloads.sourceforge.net/project/libpng/libpng12/1.2.59/libpng-1.2.59.tar.xz",
-                "sha256": "b4635f15b8adccc8ad0934eea485ef59cc4cae24d0f0300a9a941e51974ffcc7"
+                "type": "git",
+                "url": "https://github.com/pnggroup/libpng",
+                "tag": "v1.6.47"
```

Comment:
the comment clearly say something that you ignored. If it's no longer relevant there in definitely no reason to build libpng.

But given you don't update the main app, this is likely wrong.

Reply:
hmm, is that not related to the ghostpdl version used?